momento-sdk/src/main/java/momento/sdk/ScsDataGrpcStubsManager.java

```diff
@@ -48,6 +54,8 @@ final class ScsDataGrpcStubsManager implements AutoCloseable {
   ScsDataGrpcStubsManager(
       @Nonnull CredentialProvider credentialProvider, @Nonnull Configuration configuration) {
     this.deadline = configuration.getTransportStrategy().getGrpcConfiguration().getDeadline();
+    this.numGrpcChannels =
+        configuration.getTransportStrategy().getGrpcConfiguration().getMinNumGrpcChannels();
```

Comment:
astute observation!

You guessed exactly right. In an ideal world I would like to make this dynamic; keep track of how many open requests they have, and automatically add more channels as needed. But that is a bigger/riskier change that will require a lot of perf testing, so I don't want to tackle it right now.

In the future, if we add that, then this setting would become the floor rather than the exact number of channels.

lmk if that makes sense.